core/chains/evm/txmgr/eth_broadcaster_test.go

```diff
@@ -1061,7 +1061,7 @@ func getLocalNextNonce(t *testing.T, kst keystore.Eth, fromAddress gethCommon.Ad
 	n, err := kst.NextSequence(fromAddress, &cltest.FixtureChainID)
 	require.NoError(t, err)
 	require.NotNil(t, n)
-	return uint64(n)
+	return uint64(n.Int64())
```

Comment:
Can this be converted in one step?
```suggestion
	return uint64(n)
```

Reply:
thanks!